src/components/Header/index.tsx

```diff
@@ -42,6 +42,9 @@ const Header = ({ t }: { t: TFunction }) => {
         <CustomNavLinkSmall onClick={() => scrollTo("product")}>
           <Span>{t("Product")}</Span>

```

Comment:
- **Optimization**  
  The `scrollTo` function is called multiple times with the same argument. Consider storing the result in a variable to improve performance.